src/MongoDB.Driver/FindOptions.cs

```diff
@@ -109,9 +143,19 @@ public CursorType CursorType
             set { _maxTime = Ensure.IsNullOrInfiniteOrGreaterThanOrEqualToZero(value, nameof(value)); }
         }
 
+        /// <summary>
+        /// Gets or sets the min key value.
+        /// </summary>
+        public BsonDocument Min
+        {
+            get { return _min; }
+            set { _min = value; }
+        }
+
         /// <summary>
         /// Gets or sets the modifiers.
         /// </summary>
+        [Obsolete("Use individual setters instead.")]
```

Comment:
Let's say "properties" instead of "setters":

    [Obsolete("Use individual properties instead.")]

It's the property that's being deprecated, not just the setter.